www/src/pages/en/usage/trpc.md

```diff
@@ -49,7 +49,9 @@ const userRouter = t.router({
 });
 ```
 
-What does this snippet do? It's a tRPC procedure (the equivalent of a route handler in a traditional backend) that first validates the input using Zod (the same validation library that we use for [environment variables](./env-variables)), in this case making sure that the input is a string and sending an informative error if this is not the case, then calls our database using [Prisma](./prisma) and returns the user whose `id` matches the one we passed in.
+What does this snippet do?
+It’s a tRPC procedure (equivalent to a route handler in a traditional backend) that first validates the input using Zod (validation library that we use for [environment variables](./env-variables)).
```

Comment:
If you want the line to break you need a blank row between

Reply:
Not really, should I also remove the linebreaks from the file? It just helped me with the rephrasing

Reply:
> Not really, should I also remove the linebreaks from the file? It just helped me with the rephrasing

Oh you don't want the line to break, then its perfectly fine as is

Reply:
The way it looks in the deployment is fine with me. I don't want the line breaks with the higher space.
I am just worried that I disprected some contributing rules by adding the breaks in the md-file (since the single linebreaks don't change the styling)